index.src.html

```diff
@@ -274,7 +291,9 @@
     <dfn for="Credential" method export>\[[DiscoverFromExternalSource]](options)</dfn> is called
     with a {{CredentialRequestOptions}} object. It returns a {{Credential}} if one can be
     returned given the options provided, `null` if no credential is available, or an error if
-    discovery fails.
+    discovery fails. If this kind of {{Credential}} is only [=effective=] for a single use or a
+    limited time, this method is responsible for generating new [=credentials=] using a
+    [=credential source=].
```

Comment:
I wonder if it's worth defining a shorthand for "single use or a limited time"-type credentials.